### PR TITLE
coverage: add ignore nexts with explanation in near-membrane-dom

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: packages/near-membrane-dom/src/browser-realm.ts [hunk 1]
```diff
@@ -27,8 +27,10 @@ const ElementRemove = (element: Element): Element =>
     ReflectApply(ElementProtoRemove, element, emptyArray);
 const NodeAppendChild = (parent: Node, child: ChildNode): ChildNode =>
     ReflectApply(NodeProtoAppendChild, parent, [child]);
+// Because it's impossible to test whether NodeLastChild is called below,
+// ignore the function body in coverage reporting.
 const NodeLastChild = (node: Node): ChildNode =>
-    ReflectApply(NodeProtoLastChildGetter, node, emptyArray);
+    /* istanbul ignore next */ ReflectApply(NodeProtoLastChildGetter, node, emptyArray);
```
Comment (jdalton) [inline packages/near-membrane-dom/src/browser-realm.ts:33]: Why is it impossible to test ? ☝️

File: packages/near-membrane-dom/src/browser-realm.ts [hunk 2]
```diff
@@ -27,8 +27,10 @@ const ElementRemove = (element: Element): Element =>
     ReflectApply(ElementProtoRemove, element, emptyArray);
 const NodeAppendChild = (parent: Node, child: ChildNode): ChildNode =>
     ReflectApply(NodeProtoAppendChild, parent, [child]);
+// Because it's impossible to test whether NodeLastChild is called below,
+// ignore the function body in coverage reporting.
```
Comment (jdalton) [inline packages/near-membrane-dom/src/browser-realm.ts:31]: If the function is never called can it just be
```js
//    
// istanbul ignore next
```

**Review by jdalton — APPROVED** [review]
Minor nits. Looks good.

File: packages/near-membrane-dom/src/browser-realm.ts (hunk 1 above)
Comment (rwaldron) [reply packages/near-membrane-dom/src/browser-realm.ts:33]: Will do!

File: packages/near-membrane-dom/src/browser-realm.ts (hunk 2 above)
Comment (rwaldron) [reply packages/near-membrane-dom/src/browser-realm.ts:31]: You're right, it can be! This was a bit too surgical 🤣

**Pull request merged**